### PR TITLE
Update: Updated NestJS docs for TypeScript Users

### PR DESCRIPTION
**Pull request by Calm-Rock**
Updated `tracing.js` file for typescript users.

### CODE REVIEW

File: docs/instrumentation/nestjs.md [hunk 1]
```diff
@@ -46,44 +46,45 @@ Step 2. Create `tracer.ts` file<br></br>
 You need to configure the endpoint for SigNoz cloud in this file.
 
 ```js
```
Comment (makeavish) [inline docs/instrumentation/nestjs.md]: is there `ts` formatting available?

File: docs/instrumentation/nestjs.md [hunk 2]
```diff
@@ -161,44 +162,45 @@ npm install --save @opentelemetry/exporter-trace-otlp-http@^0.39.1
...
+import { SemanticResourceAttributes } from '@opentelemetry/semantic-conventions';
 
+// Configure the SDK to export telemetry data to the console
+// Enable all auto-instrumentations from the meta package
 const exporterOptions = {
-  // highlight-start
-    url: 'http://localhost:4318/v1/traces'
-    // highlight-end
-  }
+  url: 'http://localhost:4318/v1/traces',
```
Comment (makeavish) [inline docs/instrumentation/nestjs.md:179]: Missing
  // highlight-start
  // highlight-end

File: docs/instrumentation/nestjs.md [hunk 3]
```diff
@@ -46,44 +46,45 @@ Step 2. Create `tracer.ts` file<br></br>
...
 
+import { getNodeAutoInstrumentations } from '@opentelemetry/auto-instrumentations-node';
+import { OTLPTraceExporter } from '@opentelemetry/exporter-trace-otlp-http';
+import { Resource } from '@opentelemetry/resources';
+import * as opentelemetry from '@opentelemetry/sdk-node';
+import { SemanticResourceAttributes } from '@opentelemetry/semantic-conventions';
+
+// Configure the SDK to export telemetry data to the console
+// Enable all auto-instrumentations from the meta package
 const exporterOptions = {
```
Comment (makeavish) [inline docs/instrumentation/nestjs.md:59]: Missing 
  // highlight-start and end

File: docs/instrumentation/nestjs.md [hunk 4]
```diff
@@ -161,44 +162,45 @@ npm install --save @opentelemetry/exporter-trace-otlp-http@^0.39.1
...
-  // gracefully shut down the SDK on process exit
-  process.on('SIGTERM', () => {
-    sdk.shutdown()
+    [SemanticResourceAttributes.SERVICE_NAME]: '<service_name>',
+  }),
+});
+
+// initialize the SDK and register with the OpenTelemetry API
+// this enables the API to record telemetry
+sdk.start()
```
Comment (makeavish) [inline docs/instrumentation/nestjs.md]: `;` missing at end of sdk.start()

File: docs/instrumentation/nestjs.md (hunk 1 above)
Comment (Calm-Rock) [reply docs/instrumentation/nestjs.md]: both `js` and `ts` look the same. Updated it to `ts`

Comment (makeavish) [reply docs/instrumentation/nestjs.md]: ok

**Review by makeavish — APPROVED** [review]

**Pull request merged**